integration/swarm/swarm.bats

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env bats
+# *-*- Mode: sh; sh-basic-offset: 8; indent-tabs-mode: nil -*-*
+#
+# Copyright (c) 2017 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Swarm testing : This will start swarm as well as it will create and
+# run swarm replicas using Nginx
+
+# Environment variables
+DOCKER_EXE="docker"
+# Image for swarm testing
+nginx_image="gabyct/nginx"
```

Comment:
I agree with @MarioCarrilloA that we should plan to get this dockerfile into our generic repo and out of a personal account. Q. do we have the relevant dockerfile stored here already??

Reply:
@grahamwhaley we already have it at https://github.com/clearcontainers/tests/tree/master/Dockerfiles/SwarmTest